src/test/java/org/jenkinsci/plugins/gitclient/GitClientCliCloneTest.java

```diff
@@ -138,7 +139,8 @@ public void test_submodule_update_timeout_logging() throws Exception {
         assertTimeout(testGitClient, "git submodule update", largerTimeout);
     }
 
-    @Issue("JENKINS-25353")
+    @Test
+	@Issue("JENKINS-25353")
```

Comment:
Please don't do white space replacements, especially replacing leading spaces with tabs.  Refer to the [contributing document](https://github.com/jenkinsci/git-client-plugin/blob/master/CONTRIBUTING.adoc#spotbugs-checks) for the reasons white space changes are generally not accepted in this repository.  New code should follow the coding style guidelines of the [SCM API plugin](https://github.com/jenkinsci/scm-api-plugin/blob/master/CONTRIBUTING.md#code-style-guidelines).
```suggestion
    @Issue("JENKINS-25353")
```